lib/remove_bg/api_client.rb

```diff
@@ -70,16 +72,14 @@ def request_remove_bg(data, api_key)
         # Faraday v0.16 & v1.0+ support streaming, v0.17 did not (rollback release)
         if req.options.respond_to?(:on_data)
           streaming = true
-          req.options.on_data = Proc.new do |chunk, _|
+          req.options.on_data = proc do |chunk, _|
             download.write(chunk)
           end
         end
       end
 
       # Faraday v0.15 / v0.17
-      if !streaming
-        download.write(response.body)
-      end
+      download.write(response.body) unless streaming
```

Comment:
Is `streaming` the same as `!streaming`?

Reply:
@AlvesJorge `unless streaming` is the same as `if !streaming`, yes.